tests/e2e/device-plugin_test.go

```diff
@@ -119,13 +103,12 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 				ValuesOptions: values,
 				CleanupOnFail: true,
 			}
-			helmClient, err = helm.NewClientFromKubeConf(opt)
-			Expect(err).NotTo(HaveOccurred())
-			_, err = helmClient.InstallChart(ctx, &chartSpec, nil)
-			Expect(err).NotTo(HaveOccurred())
+
+			By("Installing k8s-device-plugin Helm chart")
+			_, err := f.HelmClient.InstallChart(ctx, &chartSpec, nil)
+			Expect(err).NotTo(HaveOccurred(), "Failed to install Helm chart %s", *HelmChart)
```

Comment:
It seems strange that this additional context is required. Does `err` not already contain enough information?

Reply:
is not required, I was just adding extra context. not a must, can be trimmed for sure

Reply:
It just makes the tests more verbose to write and the value of the additional context is not clear. Do you have an example of the difference with and without this?

Reply:
Agreed